android/src/main/java/io/mosip/tuvali/transfer/TransferReport.kt

```diff
@@ -11,22 +18,29 @@ class TransferReport  {
     SUCCESS
   }
 
-  constructor(type: ReportType, totalPages: Int, missingSequences: IntArray?) {
+  constructor(type: ReportType, missingSequences: IntArray, maxDataBytes: Int) {
+    val transferReportPageSize: Int = (maxDataBytes - PAGE_NUMBER_SIZE_IN_BYTES - TYPE_SIZE_IN_BYTES) / CHUNK_SEQUENCE_NUMBER_IN_BYTES
+    val missedCount = missingSequences.size
```

Comment:
Here transferReportPageSize is storing the count of number of chunks that can fit into single transfer page. I think this can be named better may be transferReportMaxChunkCount or transferReportChunkCapacity.

and also similarly missedCount can be named as missedChunksCount

Reply:
I have changed missedCount to missedSequenceNumberCount. But transferReportPageSize variable name seems fine to me as we have a concept of page in transfer report. And the variable makes sense if we think in that perspective.